src/main/proto/core/udiscovery/v3/udiscovery.proto

```diff
@@ -20,35 +20,32 @@
  */
 syntax = "proto3";
 
-package uprotocol.core.udiscovery.v2;
+package uprotocol.v1.core.udiscovery.v3;
```

Comment:
This format for package is not standard, and I recommend following conventional style

uprotocol core includes UAttributes, UUri, UUID, and these can be versioned under uprotocol.v1

service interface like udiscovery doesn't have to be under urptocol.v1 but uses core types defined in uprotocol.v1

this will have the added benefit of allowing one version of udiscovery support multiple uprotocol versions in the future (if needed)

Reply:
Fixed